### DISCUSSION TITLE
Add Sound Blaster Live!

### DISCUSSION THREAD

**Discussion by lazerode** [86Box/86Box#3669] [Ideas]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
It would be nice to add Sound Blaster Live! into 86Box.

**Describe alternatives you've considered**
_None_

**Additional context**
_None_

**Comment by Torinde** [comment]
Important missing item to emulate, completely agree.

My simplistic view: SB Live! = Ensoniq AudioPCI + AWE64 (both already emulated, so should be feasible to implement?)

Live! and Audigy/LS and Audigy 2 Value/NX/ZS source code documentation (potentially partial). https://github.com/torvalds/linux/tree/master/sound/pci/emu10k1

Related: #2795, joncampbell123/dosbox-x/issues/3597

**Comment by Torinde** [comment]
This discussion is duplicate of #2795
